ISSUES.md

```diff
@@ -0,0 +1,30 @@
+# Issue triaging
+
+## `needs author feedback`
+
+This is something that needs additional clarification from the author.
+
+Note: issues with this label are automatically marked stale if there is no activity for 7 days,
+and then closed automatically if there is no response from the author within 7 days after that.
+
+## `needs minimal repro`
+
+Minimal repros are helpful for a lot of issues (not only for bugs).
+
+They allow the maintainers (and others) to much better understand exactly what the
+issue author is trying to do, and what they expect to happen.
+
+Ideally minimal repros should be posted to a github repository,
+and should include a `README.md` that explains exactly how to build and run the repro,
+and what the output of the repro is, and what you expect (or would like) the output of
+the repro to be.
+
+Note: issues with this label are automatically marked stale if there is no activity for 30 days,
+and then closed automatically if there is no response from the author within 30 days after that.
+
+## `needs external contribution`
+
+This is something that fits the scope of the project,
+but that the maintainers aren't planning to take on themselves.
+
+Note: issues with this label are automatically closed if there is no activity for 365 days.
```

Comment:
I think this may be worth it to help control the issue backlog long-term

Reply:
I like that idea 👍